### PR TITLE
[bug] Package should be removed if the grep matches

### PR DESCRIPTION
**Pull request by kaistian**

### CODE REVIEW

File: misc/scripts/install-local.sh [hunk 1]
```diff
@@ -126,7 +126,7 @@ if echo -n "$depends" > /dev/null 2>&1; then
 fi
 if ! [[ -z $replace ]] ; then
```
Comment (wizard-28) [inline misc/scripts/install-local.sh:127]:
```suggestion
if [[ -n $replace ]] ; then
```

File: misc/scripts/install-local.sh [hunk 2]
```diff
@@ -126,7 +126,7 @@ if echo -n "$depends" > /dev/null 2>&1; then
...
     dpkg-query -W -f='${Status}' $replace 2>/dev/null | grep -q "ok installed"
-    if [[ $? -eq 1 ]] ; then
+    if [[ $? -eq 0 ]] ; then
```
Comment (wizard-28) [inline misc/scripts/install-local.sh:129]:
```suggestion
    if [[ $(dpkg-query -W -f='${Status}' $replace 2>/dev/null | grep -q "ok installed") -eq 0 ]] ; then
```

**Review by wizard-28 — CHANGES_REQUESTED** [review]
Also, fix the CodeFactor issue

**Review by Henryws — APPROVED** [review]
looks good

**Review by wizard-28 — APPROVED** [review]
I'll merge

**Pull request merged**